internal/bundle/bind_resource_test.go

```diff
@@ -124,3 +123,63 @@ func TestAccAbortBind(t *testing.T) {
 	require.NotEqual(t, job.Settings.Name, fmt.Sprintf("test-job-basic-%s", uniqueId))
 	require.Contains(t, job.Settings.Tasks[0].NotebookTask.NotebookPath, "test")
 }
+
+func TestAccGenerateAndBind(t *testing.T) {
+	env := internal.GetEnvOrSkipTest(t, "CLOUD_ENV")
+	t.Log(env)
+
+	ctx, wt := acc.WorkspaceTest(t)
+	gt := &generateJobTest{T: t, w: wt.W}
+
+	uniqueId := uuid.New().String()
+	bundleRoot, err := initTestTemplate(t, ctx, "with_includes", map[string]any{
+		"unique_id": uniqueId,
+	})
+	require.NoError(t, err)
+
+	w, err := databricks.NewWorkspaceClient()
+	require.NoError(t, err)
+
+	jobId := gt.createTestJob(ctx)
+	t.Cleanup(func() {
+		_, err = w.Jobs.Get(ctx, jobs.GetJobRequest{
+			JobId: jobId,
+		})
+		if err == nil {
+			gt.destroyJob(ctx, jobId)
+		}
+	})
+
+	t.Setenv("BUNDLE_ROOT", bundleRoot)
+	c := internal.NewCobraTestRunnerWithContext(t, ctx, "bundle", "generate", "job",
+		"--key", "test_job_key",
+		"--existing-job-id", fmt.Sprint(jobId),
+		"--config-dir", filepath.Join(bundleRoot, "resources"),
+		"--source-dir", filepath.Join(bundleRoot, "src"))
+	_, _, err = c.Run()
+	require.NoError(t, err)
+
+	_, err = os.Stat(filepath.Join(bundleRoot, "src", "test.py"))
+	require.NoError(t, err)
+
+	matches, err := filepath.Glob(filepath.Join(bundleRoot, "resources", "test_job_key.yml"))
+	require.NoError(t, err)
+
+	require.Len(t, matches, 1)
+
+	c = internal.NewCobraTestRunner(t, "bundle", "deployment", "bind", "test_job_key", fmt.Sprint(jobId), "--auto-approve")
+	_, _, err = c.Run()
+	require.NoError(t, err)
+
+	err = deployBundle(t, ctx, bundleRoot)
+	require.NoError(t, err)
+
+	err = destroyBundle(t, ctx, bundleRoot)
+	require.NoError(t, err)
+
+	// Check that job is unbound and exists after bundle is destroyed
```

Comment:
I suspect the reverse is true here; the job will have been bound and destroyed.

Reply:
Indeed, a wrong comment copy